core/src/main/scala/org/apache/spark/h2o/backends/external/ExternalH2OBackend.scala

```diff
@@ -51,6 +52,16 @@ class ExternalH2OBackend(val hc: H2OContext) extends SparklingBackend with Exter
       "-disown"
     )
 
+    conf.sslConf match {
+      case Some(ssl) => {
+        cmdToLaunch ++ Array("-internal_security_config", ssl)
```

Comment:
Ah I see. So basically users need to copy the config file to node with sparkling water ?

I think we can just comment this behaviour for now and explain what needs to be done if user wants to connect to established h2o cluster. I expect anyway that most of the users will be using external sparkling water with h2o started on yarn automatically, as it is just way easier than having to configure and start your own h2o cluster 👍